src/components/Chip/__tests__/Chip.spec.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import toJson from 'enzyme-to-json';
+import { create } from 'react-test-renderer';
 import { Chip } from '../Chip';
 
 describe('<Chip> that renders a pill shaped chip', () => {
     it('should render correctly', () => {
-        const wrapper = shallow(<Chip>some text</Chip>);
-        expect(toJson(wrapper)).toMatchSnapshot();
+        const wrapper = create(<Chip>some text</Chip>);
+        expect(wrapper).toMatchSnapshot();
```

Comment:
do we need to convert to JSON by using toJSON() method?

Reply:
No, here we use new library it doesn't require it here you can check some example https://github.com/textkernel/oneui/pull/906/files